codice/ddf
# DDF-2329 - add logback-access config to paxweb fragment and consolida…

…ted platform-security-session
#### What does this PR do?
Updates DDF to use Logback access logging for Common Log Format (CLF) or HTTP Access logging instead of the jetty NCSA logs.
Jetty has a bug that is resolved in 9.3, that the content-length is not reported correctly in the NCSA logs, which is the reason for this change.
The HTTP Access logs can now be enable by setting  `org.codice.ddf.http.access.log.enabled=true` in etc/system.properties.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
 @troymohl @emmberk @millerw8 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl 
@kcwire
#### How should this be tested?
Unzip. Change `org.codice.ddf.http.access.log.enabled=true` in etc/system.properties. Start DDF. Verify the http access logs are written to DDF_HOME/logs/*.log with the content-length.
#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-2329
#### Screenshots (if appropriate)
#### Checklist:
